core/src/main/java/com/xmartlabs/bigbang/core/helper/IoSchedulersTransformationHelper.java

```diff
@@ -0,0 +1,81 @@
+package com.xmartlabs.bigbang.core.helper;
+
+import android.support.annotation.CheckResult;
+import android.support.annotation.NonNull;
+
+import io.reactivex.CompletableTransformer;
+import io.reactivex.FlowableTransformer;
+import io.reactivex.MaybeTransformer;
+import io.reactivex.ObservableTransformer;
+import io.reactivex.Single;
+import io.reactivex.SingleTransformer;
+import io.reactivex.schedulers.Schedulers;
+
+/**
+ * Contains the different Io Schedulers transformations for main RxJava operators.
+ */
+public class IoSchedulersTransformationHelper {
```

Comment:
What do you think of `SchedulersTransformation` or `SchedulersTransformationHelper`. So in the future we could include another transformation